internal/gatewayapi/backendtrafficpolicy.go

```diff
@@ -557,10 +554,17 @@ func (t *Translator) translateBackendTrafficPolicyForGateway(policy *egv1a1.Back
 			if policy.Spec.UseClientProtocol != nil {
 				setIfNil(&r.UseClientProtocol, policy.Spec.UseClientProtocol)
 			}
+
+			if errs != nil {
```

Comment:
this needs to be moved up, right below `if r.Traffic != nil`

Reply:
Done 👍🏽